server/fleet/app.go

```diff
@@ -287,6 +288,16 @@ func (d *Duration) UnmarshalJSON(b []byte) error {
 	}
 }
 
+type MDMSettings struct {
```

Comment:
@gillespi314 I just realized that I left this here 😅 , let's keep using `MDM` in this PR and we can tackle the rename separately.

Probably a good idea to move the comment too.